### ISSUE TITLE
How can you use the language?

### ISSUE THREAD

**Issue by VenkSociety** [davidcallanan/py-myopl-code#2]
Trying to get `example.myopl` to run but I've no idea how. Is there a specific command to execute via shell CLI? Or a way to get it to run through the Python CLI?

**Comment by aideng1694** [comment]
## TL;DR
```myopl
RUN("example.myopl")
```
The above code is how to run the example file in the [`ep14`][ep14] directory

## General Explanation
The code below is how to run a file from anyware

This is what I typed in my terminal:
```bash
me@aideng1694:~/Projects/Language/src$ python3 ep14/shell.py 
basic > RUN("ep14/example.myopl")
Greetings universe!
loop, spoop
loop, spoop
loop, spoop
loop, spoop
loop, spoop
0
```

What this code does:
  - Runs [episode 14's version][ep14] of [shell.py][ep14/shell.py] using Python 3 (*In my case, Python* ***3.8***)
  - Executes `RUN` with the path "[ep14/example.myopl]" (*The path to the* `example.myopl` *file*)
  - Executes the contents of "[ep14/example.myopl]"

[ep14]: https://github.com/davidcallanan/py-myopl-code/tree/master/ep14
[ep14/shell.py]: https://github.com/davidcallanan/py-myopl-code/tree/master/ep14/shell.py
[ep14/example.myopl]: https://github.com/davidcallanan/py-myopl-code/tree/master/ep14/example.myopl
